nodejs/awsx/apigateway/api.ts

```diff
@@ -404,6 +419,28 @@ interface SwaggerOperation {
     responses?: { [code: string]: SwaggerResponse };
     "x-amazon-apigateway-integration": ApigatewayIntegration;
     "x-amazon-apigateway-request-validator"?: reqvalidation.RequestValidator;
+
+    /**
+     * security maps to the authorizerName defined in the top level securityDefinitions
+     */
```

Comment:
i'm not sure i understand this comment.  by saying "maps to" it sounds lke the authorizerName will be in the `string[]` value of the map.  But it looks more like the authorizerName is the key.  So i would think it would be "maps from".  (also, i don't know what the `string[]` means here.

Reply:
i will try to clarify here.

Reply:
> The authorizerNames' values are empty arrays.

Wacky.  So nothing ever goes in there?

Reply:
ah gotcha